src/Service/SF1500/SF1500.php

```diff
@@ -116,27 +114,30 @@ public function getOrganisationFunktionerFromUserId(string $brugerId)
      */
     public function soegOrganisationFunktioner(?string $brugerId, ?string $funktionsNavn, ?string $organisationId, ?string $funktionsTypeId)
     {
-        $token = $this->fetchSAMLToken();
-
-        $data = $this->organisationFunktionSoeg($brugerId, $funktionsNavn, $organisationId, $funktionsTypeId, $token);
+        $data = $this->organisationFunktionSoeg($brugerId, $funktionsNavn, $organisationId, $funktionsTypeId);
 
         $idListeKeys = [
             'ns3SoegOutput',
             'ns2IdListe',
             'ns2UUIDIdentifikator',
         ];
 
-        return $this->getValue($data, $idListeKeys);
+        $ids = $this->getValue($data, $idListeKeys, []);
+
+        // If only one result is found convert it into array with one entry.
+        if (is_string($ids)) {
+            $ids = [$ids];
+        }
+
+        return $ids;
```

Comment:
This can be written as `return (array)$this->getValue($data, $idListeKeys, []);` (cf. https://www.php.net/manual/en/language.types.array.php#language.types.array.casting).

Reply:
👍